swarm/src/behaviour.rs

```diff
@@ -293,6 +293,15 @@ pub enum NetworkBehaviourAction<TInEvent, TOutEvent> {
         /// relative to other observed addresses.
         score: AddressScore,
     },
+
+    /// Instructs the `Swarm` to initiate a graceful close of the connection
+    /// with a peer.
+    DisconnectPeer {
+        /// The peer to disconnect.
+        peer_id: PeerId,
+        /// The ID of the connection whose `ProtocolsHandler` to disconnect.
```

Comment:
These docs seems to be slightly off given that `DisconnectPeerHandler` is not just the ID.

Reply:
I see, thank you!

Reply:
```suggestion
        /// Whether to close a specific or all connections to the given peer.
```